include/uv.h

```diff
@@ -244,6 +244,11 @@ typedef enum {
 UV_EXTERN unsigned int uv_version(void);
 UV_EXTERN const char* uv_version_string(void);
 
+typedef void* (*uv_malloc_func)(size_t size);
+typedef void (*uv_free_func)(void* ptr);
+
+UV_EXTERN int uv_replace_allocator(uv_malloc_func malloc_func, uv_free_func free_func);
```

Comment:
Style nit: long line, please put the second argument on a new line and make it line up with the first.  (Or I can do that before I land it.)